apps/src/applab/ai.js

```diff
@@ -67,52 +67,6 @@ function generateCodeDesignElements(modelId, modelData) {
       var addFeature = `data.${alphaNumFeature} = getText("${fieldId}");`;
       inputFields.push(addFeature);
     });
-  } else {
```

Comment:
Do we still need the corresponding `if` above?

Reply:
Not necessarily.  It protects against a scenario where `modelData.features` is for some reason undefined, so we don't hit a  `Cannot read property 'forEach' of undefined` error, but I don't really think there's a likely path we'll get into that situation. I'll remove.